internal/cmd/list_test.go

```diff
@@ -75,6 +76,19 @@ func TestListCmd(t *testing.T) {
 	})
 	assert.NilError(t, err)
 
+	for _, uniqueID := range []string{"space", "moon", "maintain"} {
+		// set client.Headers so each destination becomes connected
```

Comment:
I had this problem as well in #3391, I thought it might make sense to have `CreateDestination` mark it as "last seen at", like this: https://github.com/infrahq/infra/pull/3391/files#diff-487038165470a56e27ec73856c4cffb0b0808e9d00fd6a3481dd59d24f4e150c